modules/nw-metallb-operator-limit-speaker-to-nodes.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-operator-install.adoc
+
+[id="nw-metallb-operator-limit-speaker-to-nodes_{context}"]
+= Limit speaker pods to specific nodes
+
+By default, when you start MetalLB with the MetalLB Operator, the Operator starts an instance of a `speaker` pod on each node in the cluster.
+You can configure the `MetalLB` custom resource with a node selector to specify which nodes run the `speaker` pods.
+
+The most common reason to limit the `speaker` pods to specific nodes is to ensure that only nodes with network interfaces on specific networks advertise load balancer IP addresses.
```

Comment:
I'd add a premise mentioning that metallb advertises only the nodes with a running speaker as destinations of the load balancer ip.